### PR TITLE
docs: fixed typo in view-parser

### PR DESCRIPTION
**Pull request by sammyskills**
**Description**
Fixed typo in view parser docs for:
- class reference for `renderString()`
- custom filters
- PHP native functions as filters

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/outgoing/view_parser/023.php [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <?php
 
-return $parser->render('myview');
+return $parser->renderString($template);
```
Comment (kenjis) [inline user_guide_src/source/outgoing/view_parser/023.php:3]: `$template` is not defined. It seems better to show working code.

Comment (sammyskills) [reply user_guide_src/source/outgoing/view_parser/023.php:3]: You are correct.

**Comment by sammyskills** [comment]
There is something I observed in the [docs](https://codeigniter4.github.io/CodeIgniter4/outgoing/view_parser.html#php-native-functions-as-filters) for PHP native functions as filters...

> and its value is any valid native PHP function prefixed with:

The function actually works with/without the backslash `\` prefix. Should this be indicated in the docs too?

**Comment by paulbalandan** [comment]
The `\` there is to indicate you want the native `str_repeat` function PHP offers. This works in your case with or without the backslash because there are no user-defined function named `str_repeat`.

**Comment by sammyskills** [comment]
Then, I suggest it should be clearly stated in the docs, something like a "note", maybe something like this:
> The `\` prefix is required if there is a user-defined function named exactly the same as the PHP native function you want to use as a filter.

WDYT?

**Comment by kenjis** [comment]
The note is more confusing.

> a user-defined function named exactly the same as the PHP native function

What do you mean by exactly the same? You cannot define `\str_repeat()`,
but can define `\App\Functions\str_repeat()`.

All PHP native functions are defined in the global namespaces.
So adding `\` is harmless.

**Comment by kenjis** [comment]
No, @sammyskills is correct.

> The function actually works with/without the backslash \ prefix.

Yes. In this context, the function name is a callable.
https://github.com/codeigniter4/CodeIgniter4/blob/411a9bbbb9c9aee8df5c9a17eee5f1d21b2d0438/system/View/Parser.php#L607-L608

The leading `/` is not needed. It is interpreted as a full qualified function name.
See https://3v4l.org/JArO7

File: user_guide_src/source/outgoing/view_parser.rst [hunk 2]
```diff
@@ -435,8 +435,8 @@ PHP Native functions as Filters
 -------------------------------
 
 You can use native php function as filters by editing **app/Config/View.php** and adding new entries to the
-``$filters`` array.Each key is the name of the native PHP function is called by in the view, and its value is any valid native PHP
-function prefixed with:
+``$filters`` array. Each key is the name which the native PHP function is called by in the view, and its value is any valid native PHP
+function prefixed with ``\``:
```
Comment (kenjis) [inline user_guide_src/source/outgoing/view_parser.rst:439]:
```suggestion
``$filters`` array. Each key is the name which the PHP function is called by in the view, and its value is any valid PHP function:
```

**Comment by kenjis** [comment]
The section title *PHP Native functions as Filters* is not accurate.
https://codeigniter4.github.io/CodeIgniter4/outgoing/view_parser.html#php-native-functions-as-filters
Because it does not matter whether it is a native function or not.

**Comment by sammyskills** [comment]
> The section title _PHP Native functions as Filters_ is not accurate. 

That means the section will have to be removed completely, and the `str_repeat` example added to the examples for custom filters.

**Comment by kenjis** [comment]
Yes, all values are PHP callables.

And the first sample code is not good.
```php
    public $filters = [
        'abs'        => '\CodeIgniter\View\Filters::abs',
        'capitalize' => '\CodeIgniter\View\Filters::capitalize',
    ];
```
`\CodeIgniter\View\Filters::abs` does not exist.
`'capitalize' => '\CodeIgniter\View\Filters::capitalize',` are already defined in `CodeIgniter\Config\View`.

**Review by kenjis — APPROVED** [review]
LGTM! Thank you.

**Review by paulbalandan — APPROVED** [review]

**Review by TimexPeachtree — APPROVED** [review]
Awesome

**Pull request merged**